homeassistant/components/roomba/strings.json

```diff
@@ -1,21 +1,39 @@
 {
   "config": {
     "step": {
-      "user": {
-        "title": "Connect to the device",
-        "description": "Currently retrieving the BLID and password is a manual process. Please follow the steps outlined in the documentation at: https://www.home-assistant.io/integrations/roomba/#retrieving-your-credentials",
+      "init": {
+        "title": "Automaticlly connect to the device",
+        "description": "Select a Roomba or Braava.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "manual": {
+        "title": "Manually connect to the device",
+        "description": "No Roomba or Braava have been discovered on your network. The BLID is the portion of the device hostname after `iRobot-`. Please follow the steps outlined in the documentation at: https://www.home-assistant.io/integrations/roomba/#retrieving-your-credentials",
```

Comment:
Urls should be put in description placeholders so the translation doesn't translate those.